### DISCUSSION TITLE
[Weekly Meeting] [2024-05-10] - Meeting Notes

### DISCUSSION THREAD

**Discussion by porcelli** [apache/incubator-kie-issues#1184] [Weekly Meetings]
### Community
 - SonataFlow logo
    - @ricardozanini propose briefly discuss it... will put to vote.
    - @baldimir suggested to review all logos.
    - @ricardozanini mentioned that projects under single umbrela, but different projects.
    - @porcelli mentioned that Apache KIE proposal insisted that we were ONE thing, not different pieces. This needs to be addressed. @ricardozanini and @tiagobento and agree that we need to have discussion how best work. @porcelli mentioned that if we had a log4j issue today, we would take more than a month to address giving our current infra and configurations.
    - @gitgabrio a bit confused about the `Apache KIE` being one thing.
    - @mariofusco let's prioritize the release! +1 from @porcelli and everything seems to agree to that.
    - @wmedvede if need review all logos, this should have brought in the beginning. @baldimir actually did.

---
### KIE 10 Release
- [Board updated](https://github.com/orgs/apache/projects/295/views/7?sliceBy%5Bvalue%5D=10.0.0)
- Compliance
  - [WIP] Rat tool found new files
- Quarkus 3.8.4
  - PR on `kie-tools` waiting for upstream fixes on Kogito Apps
    - https://github.com/apache/incubator-kie-tools/pull/2272
     - Issues found that has been addressed by a series of PRs fixing around postgresql embedded and quarkus config files
     - issue with embedded postgres setting data source config, adjusting configs is solving the issue and this has been addresses in several PRs across runtime repos. upgraded embedded postgres to latest that before was also breaking
- [Circular dependency](https://github.com/apache/incubator-kie-issues/issues/967)
     - https://github.com/apache/incubator-kie-issues/issues/964
     - Not blocking release itself, but needs to be addressed to publish to Maven Central
- https://github.com/apache/incubator-kie-issues/issues/1042
   - 999-SNAPSHOT depedency - @mariofusco asking for clarifications
   - @tiagobento is clarifying that this property is not updated in anyplace... so in a release this would be kept 999-SNAPSHOT
   - @porcelli will address this to sue project.vesion and it should fix the issue.
   - @baldimir mentioned that there are other maven plugins to address this issue
   - @tiagobento mentions that this is part of the conversation of release procedure
- Following the [Proposal](https://lists.apache.org/thread/pw327lkpmy9gxklq4t5zbwzxjo2y3c0w) - we are ready for branch once we have the above done!

---

### Release procedures

- Go libs release procedures (impacting kogito-serverless-operator and kn-plugin-workflow)
   - @tiagobento asking a bit more clarifications on how go modules are consumed by external. It might need change source code to be able to consumable externally if this is needed.
   - @ricardozanini no need to worry about this now. for Go it's required to have a tag. 
   - @tiagobento kogito operator can't be used because is not part of the release. we can't tamper with new commit the voted release. @tiagobento suggesting if we could skip for 10.
   - @ricardozanini for future there's going to have libs exposed, but for 10 it's not a required now.
- Update jitexecutor-native publication jobs on `kogito-apps`.
  - Waiting Apache's infra team to configure secrets with Maven repository tokens for GHA workflow (required due to macOS and Windows being necessary, and Quarkus native-image compilation not supporting cross-compilation and macOS and Windows not existing on Apache's Jenkins)
- Documented drafted [here](https://docs.google.com/document/d/1t-08b970i9PMBy-TSM64x0RUH46pzwnIhEi-LVeL0lg/edit?usp=sharing)
     - Missing information
         - how to build and upgrade versions of Kogito Images
         - what profiles to use for each repository (drools, optaplanner, kogito-runtimes, kogito-apps)

-----
Recording

https://us06web.zoom.us/rec/share/pxqM3pr-geEnD9gOy-ezFYzhvc1WOXq01ZABYBgQjdfJLzdPRoo62G7JTKvqTyrg.9yO7ixe1LT-n98kS
Passcode: L*6X&rw?

**Comment by tiagobento** [comment]
Btw here's the ticket for the `jitexecutor-native` changes
-  https://github.com/apache/incubator-kie-issues/issues/1191
